### ISSUE TITLE
Documentation for Binary Log Protocol Client

### ISSUE THREAD

**Issue by zenflow** [sidorares/node-mysql2#688]
There currently seems to be no documentation of the Binary Log Protocol Client feature...

I'm not there yet, but I will need to know about this feature. 

When I do get there, I'll start posting comments here with what I find out.

Please, anyone in the same situation do the same.

At some point these details can be collected and a PR submitted to update documentation.

**Comment by sidorares** [comment]
Yes, binlog client examples / docs are very much needed. If you want to add them please do ( maybe collaborate with @shtse8 on that
